test/unit/helpers/QtiTestSanitizerTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\test\unit\helpers;
+
+use oat\generis\test\TestCase;
+use oat\taoQtiTest\helpers\QtiTestSanitizer;
+
+class QtiTestSanitizerTest extends TestCase
+{
+    /**
+     * @dataProvider sanitizeContentProvider
+     */
+    public function testSanitizeContent(string $content, string $expected): void
+    {
+        $this->assertEquals($expected, QtiTestSanitizer::sanitizeContent($content));
+    }
+
+    public function sanitizeContentProvider(): array
```

Comment:
Good set of tests. Kind of overlaps with https://github.com/oat-sa/tao-core/blob/master/test/unit/helpers/DisplayTest.php#L50 which is not a problem, but I suppose the only thing we do here that is not covered there is htmlspecialchars_decode...